e2e-tests/steps/apply.ts

```diff
@@ -117,8 +117,12 @@ export const addNote = async (page: Page) => {
   await expect(page.locator('.moj-timeline__title').first()).toContainText('Note')
 }
 
-export const viewApplicationFromPrisonDashboard = async (applicationId: string, page: Page) => {
+export const goToPrisonDashboard = async (page: Page) => {
   await page.goto(`/applications/prison`)
-  await page.locator(`a[href="/applications/${applicationId}/overview"]`).click()
+}
+
+export const viewApplicationMadeByAnotherUser = async (page: Page, name: string) => {
+  const tableRow = page.locator('.govuk-table__row').filter({ hasNotText: name }).last()
```

Comment:
Would it give us better test feedback if we first had another expectation which said "expect table to include the authors name"?

Reply:
that's a good idea - that should work if it's always run after the 'create an application' test, which it should do on CI.  The only time it might be flakey is running it locally, if we are running the tests out of order there may not be a recent application made by the user, but I think the trade off is we are getting more coverage on CI.

Reply:
done